js-intro.js

```diff
@@ -55,7 +95,18 @@ var notAString2 = 42
 
 var toonimals = [ {name: "Itchy", animal: "mouse"}, {name: "Stimpy", animal: "cat"}, {name: "Daffy", animal: "duck"}, {name: "Scratchy", animal: "cat"}, {name: "Ren", animal: "dog"}, {name: "Felix", animal: "cat"}]
 
-
+const animalFinder = (array) => {
+    let newArray = array.filter(value => value.animal === "cat")
+    return (newArray)
+}
+console.log(animalFinder(toonimals));
 
 // --------------------7) Using the toonimals variable from #6, create a function that returns only the names of the non-cats.
 // Expected output: "Itchy" "Daffy" "Ren"
+
+const animalFinder2 = (array) => {
+    let newArray = array.filter(value => value.animal !== "cat")
```

Comment:
Consider naming your variables more precisely. For example, instead of `newArr` 1 & 2, you might name them `justCats` and `catNames`. This will just add further clarity to your code and is a good habit to form.